task-09/README.md

```diff
@@ -0,0 +1,58 @@
+[previous task](../task-08)
+
+# Task 9: RIOT and RPL
+
+This task will demonstrate basic routing functionalities in RIOT provided by the [RPL](https://tools.ietf.org/html/rfc6550) (IPv6 Routing Protocol for Low-Power and Lossy Networks) routing protocol and multihop communication.
+
+It uses an adapted version of the default [`gnrc_networking`](https://github.com/RIOT-OS/RIOT/tree/master/examples/gnrc_networking) example in the RIOT repository.
+
+## Prerequisites
+
+* Compile the application and run on your `BOARD`
+* For this task you need at least three boards (you can form groups with your neighbors)
+* Each "group" should change to an individual RF channel. It can be changed by e.g.
+
+```
+ifconfig <if_id> set chan 11
+
+```
+
+Use the `ifconfig` command to obtain the `<id_id>`.
+
+* Check if this worked by looking at the output of `ifconfig` again. Also have a look at the preconfigured IPv6 addresses that all have a local scope.
+
+
+## Task 9.1: Initialize RPL
+
+* In order to use RPL we have to choose **one** RPL root node and configure a global IPv6 address for it. This is simply done by adding the address to the network interface (`<if_id>`) via `ifconfig`:
+
+```
+ifconfig <if_id> add 2001:db8::1
+```
+
+* Next we need to initialize RPL on the network interface (`<if_id>`) for **all** nodes: 
+
+```
+rpl init <if_id>
```

Comment:
you don't need this when including in your makefile: `USEMODULE += auto_init_gnrc_rpl`